xtext-website/documentation/103_domainmodelnextsteps.md

```diff
@@ -339,7 +339,39 @@ This tutorial is about testing the parser, the linker and the validator for the
     }
     ```
 
-3.  After saving the Xtend file, it is time to run the test. Select *Run As &rarr; JUnit Test* from the editor's context menu.
+3. The [CompilationTestHelper]({{site.src.xtext_extras}}/org.eclipse.xtext.xbase.testing/src/org/eclipse/xtext/xbase/testing/CompilationTestHelper.java) utility class comes in handy while unit testing the custom generator implementations:
+
+	```xtend
+	@Inject extension CompilationTestHelper
+
+	@Test def test() {
+		'''
+			datatype String
+			
+			package my.company.blog {
+				entity Blog {
+					title: String
+				}
+			}
+		'''.assertCompilesTo('''
+			package my.company.blog;
+
+			public class Blog {
+				private String title;
+				
+				public String getTitle() {
+					return title;
+				}
+				
+				public void setTitle(String title) {
+					this.title = title;
+				}
+			}
+		''')
+		}
+```
+
+4.  After saving the Xtend file, it is time to run the test. Select *Run As &rarr; JUnit Test* from the editor's context menu.
```

Comment:
Probably should be changed to 'run the test**s**' now

Reply:
Fixed.